desktop/src/main/java/bisq/desktop/main/dao/governance/GovernanceView.java

```diff
@@ -177,6 +173,13 @@ private void loadView(Class<? extends View> viewClass) {
         else if (view instanceof ProposalsView) toggleGroup.selectToggle(open);
         else if (view instanceof VoteResultView) toggleGroup.selectToggle(result);
     }
+
+    private void setupOpenMenuButtonText(DaoPhase.Phase phase) {
+        if (phase == DaoPhase.Phase.BLIND_VOTE)
+            open.setLabelText(Res.get("dao.proposal.menuItem.vote"));
+        else
+            open.setLabelText(Res.get("dao.proposal.menuItem.browse"));
+    }
```

Comment:
As we had issues in the past concerning using no brackets for one line conditionals we decided to use always {}

Reply:
Fixed